### PR TITLE
Add icons for severity handling in notification manager

### LINKED ISSUE #3732: ✨ Warnings appear more "alarming" than error messages
### Overview of design need
In napari's notification windows, error messages appear less dramatic than warnings. A user could easily miss that a problem exists.

When python raises an error, the notification looks rather un-dramatic in napari:
image

If a warning is shown, it looks more dramatic:
image

Would it be possible to replace the grey error icon with something red? E.g. a red stop sign is quite typical for error messages on windows:

image

### What level of design is needed? (Choose all that apply)
_This section may be updated by the designer / UX researcher working on this issue_

- [ ] **User experience research:**  high-level recommendation/exploration of user needs, design heuristics, and / or best practices to inform a design experience  
(Use this option when you feel there’s a challenge to be solved, but you’re curious about what the experience should be — may involve research studies to understand challenges/opportunities for design)
- [ ] **information flow / conceptual:** organizing and structuring of information flow and content, including layout on screen or across multiple steps
- [x] **Visual:** creating mockups, icons, etc
(If choosing this level alone, it means that the content to be mocked up and its organization is already known and specified)

### Is design a blocker?
- [x] **Yes:** engineering cannot proceed without a design first
- [ ] **No:** engineering can create a first version, and design can come in later to iterate and refine

If selecting **Yes**, how much design input is needed to unblock engineering? For example, is a full, final visual design needed, or just a recommendation of which conceptual direction to go?

We just need another icon

### PR DESCRIPTION
**Pull request by goanpeca**
# Description

## Type of change

- [x]  Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

Fixes #3732

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

Screen Shot 2021-12-06 at 10 29 51 PM
Screen Shot 2021-12-06 at 10 30 34 PM

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Comment by goanpeca** [comment]
@haesleinhuepf could you test this one :) ?

**Review by tlambert03 — APPROVED** [review]
nice!

File: napari/_qt/dialogs/qt_notification.py [hunk 1]
```diff
@@ -108,6 +109,28 @@ def __init__(
...
+        default_color = getattr(get_theme(theme, False), 'icon')
+
+        # FIXME: Should these be defined at the theme level?
+        # Currently there is a warning one
+        colors = {
+            'error': "#D85E38",
+            'warning': "#E3B617",
+            'info': default_color,
+            'debug': default_color,
+            'none': default_color,
```
Comment (goanpeca) [inline napari/_qt/dialogs/qt_notification.py:128]: Should we add/create other icons for info (there is an existing one already) and debug @isabela-pf ?

**Review by sofroniewn — APPROVED** [review]
Looks beautiful! Thanks!!

**Comment by haesleinhuepf** [comment]
Thanks @goanpeca ! The new icons look awesome 🤩

File: napari/_qt/dialogs/qt_notification.py (hunk 1 above)
Comment (isabela-pf) [reply napari/_qt/dialogs/qt_notification.py:128]: I know this has already been merged, but are you asking about creating icons for `info`, `debug`? I can if there is interest, I would just want to check that they are used the ways the name imply.

However I also know there's been talk about an icon overhaul in issues like #3136, so I'm not sure if it's more or less helpful to do this work when we aren't looking at it from the big picture. If other people agree now's a good time to add in these other icons, I'm happy to make it happen.

Comment (goanpeca) [reply napari/_qt/dialogs/qt_notification.py:128]: @napari/core-devs  :) ?

**Pull request merged**